tests/test_webhooks.py

```diff
@@ -640,15 +649,16 @@ def test_riemann_webhook(self):
     def test_stackdriver_webhook(self):
 
         # open alert
+        import nose.tools; nose.tools.set_trace()
```

Comment:
You need to remove this line and the line below and the tests will pass. Thanks.